torch/_tensor_str.py

```
@@ -127,12 +128,33 @@ def tensor_totype(t):
     return t.to(dtype=dtype)
 
 
+@lru_cache(maxsize=256)
+def _get_tensor_format_config(dtype_str: str, device_str: str, numel: int) -> tuple[bool, int]:
```

Ruff (0.14.6)

Unused function argument: `dtype_str`

(ARG001)